unit-tests/src/test/scala/java/util/FormatterSuite.scala

```diff
@@ -2924,6 +2924,14 @@ object FormatterSuite extends tests.Suite {
     }
   }
 
+  // Next test uses, by design, Locale.GERMAN and will continue to fail
+  // even after PR #1298 and this PR for issue ##1296 are merged.
```

Comment:
Changes made as requested.  

Since this PR requires #1298, Later parts of Travis CI fail. Scalafmt & tools pass.
The failure should go away once #1298 is merged.